step.py

```diff
@@ -16,15 +16,12 @@ def getJson(url, queryParams):
 
   return json.load(response)
 
-def getGif(gifQuery):
+def getGif(gifQuery, apiKey):
   baseUrl = 'http://api.giphy.com/v1/gifs/search'
```

Comment:
Should this be https?

Reply:
Yes, you're right. Official API explorer uses https. i.e. `https://api.giphy.com/v1/gifs/search`